src/features/user/ManageProjects/ProjectsContainer.tsx

```diff
@@ -108,7 +108,10 @@ function SingleProject({ project }: any) {
       <div className={styles.projectInformation}>
         <p className={styles.projectName}>{project.name}</p>
         <p className={styles.projectClassification}>
-          {project.classification} •{' '}
+          {project.purpose === 'trees'
+            ? project.classification
+            : project.metadata.ecosystems}{' '}
```

Comment:
Add null check for metadata

![image](https://user-images.githubusercontent.com/37466979/170185514-92ae4e13-ae68-4a87-ab90-870c10f9ac9c.png)

Reply:
it have been fixed now